Copy the right ramoops logs to the crashlog folder

jiaxuan-guo opened the pull request:
Crashlog should save console-ramoops and dmesg-ramoops after kernel panic happens.

Tracked-On: OAM-110472

sysopenci commented:
Jira issue not in ['bug','improvement'],
    make sure jira issue is of type bug or improvement.

jiaxuan-guo committed:
Copy ramoops logs to the crashlog folder

Crashlog should save console-ramoops and dmesg-ramoops after kernel panic happen.

Tracked-On: OAM-110472
Signed-off-by: Jade Guo <jade.guo@intel.com>

sysopenci commented:
Jira issue not in ['bug','improvement'],
    make sure jira issue is of type bug or improvement.

sysopenci commented:
Jira issue not in ['bug','improvement'],
    make sure jira issue is of type bug or improvement.

jingdlu reviewed (approved):
LGTM

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_smr0_master-preintegration/builds/145

Done by "Guo, Jade" <jade.guo@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon_smr0_master-premerge/builds/80

Done by "Guo, Jade" <jade.guo@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Guo, Jade" <jade.guo@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon_smr0_master-mergerequest/builds/84
